data.sql

```diff
@@ -1,16 +1,21 @@
 /* Populate database with sample data. */
 
+INSERT INTO owners (full_name, age)
+VALUES ('Sam Smith', 34), ('Jennifer Orwell', 19), ('Bob', 45),
+('Melody Pond', 77), ('Dean Winchester', 14), ('Jodie Whittaker', 38);
 
-INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, WEIGHT_KG) 
-VALUES ('Agumon', '2020-02-03', 0, true, 10.23);
-INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, WEIGHT_KG) 
-VALUES ('Gabumon', '2018-11-15', 2, true, 8);
-INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, WEIGHT_KG) 
-VALUES ('Pikachu', '2021-01-07', 1, false, 15.04);
-INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, WEIGHT_KG) 
-VALUES ('Devimon', '2017-05-12', 5, true, 11);
-INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, WEIGHT_KG) 
-VALUES ('Charmander', '2020-02-08', 0, false, -11), ('Plantmon', '2022-11-15', 2, true, -5.7),
-('Squirtle', '1993-04-02', 3, false, -12.13), ('Angemon', '2005-06-12', 1, true, -45),
- ('Boarmon', '2005-06-07', 7, true, 20.4),
-('Blossom', '1998-10-13', 3, true, 17);
\ No newline at end of file
+INSERT INTO species (name) VALUES ('Pokemon'), ('Digimon');
+
```

Comment:
**Changes Required**
We are missing the following `UPDATES`
- [ ] if an animal's name ends with mon it should have Digimon as a species_id, all other animals should have Pokemon instead.
- [ ] all animals got assigned owners.
- [ ]

Reply:
Hi @oxenprogrammer, thank you for the feedback. All animals have an owner and I modified the animal's data to include the species of the animals. Kindly check the data.sql file for that. The project requirements say that we should modify the inserted data and not use updates

![image](https://user-images.githubusercontent.com/77106962/144390740-a9063ee6-0076-4879-a200-dcbc829c0cb0.png)

### Kindly advise me on what to do. 🙏